lib/joken/application.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Joken.Application do
+  @moduledoc """
+  Application behaviour for Joken. Used to bootstrap some key options.
+  """
+
+  def start(_type, _args) do
```

Comment:
Since there is an application, is it possible to pass in configuration here instead of using the config file?

Reply:
Not sure what you mean here. Do you mean passing configuration through release files?